contents/docs/alerts/index.mdx

```diff
@@ -0,0 +1,128 @@
+---
+title: Alerts
+sidebar: Docs
+showTitle: true
+availability:
+  free: partial
+  selfServe: full
+  enterprise: full
+---
+
+
+Alerts enable you to monitor your [insights](/product-analytics) and get notified when the value of the insight breaches a threshold that you've set (for example, number of unique visitors to your site increased by 5% week over week).
+Currently alerts are supported on all [trends](/docs/product-analytics/trends) (support for [funnels](/docs/product-analytics/funnels) coming soon!).
+
+
+## How to create an alert?
+1. Alerts are based off of [insights](/product-analytics). You can pick any existing trend insight or create a new one.
+2. Click on the "Alerts" button on top right of the insight page. This shows you all the alerts you've setup for this insight.
```

Comment:
```suggestion
2. Click on the **Alert** button on top right of the insight page. This shows you all the alerts you've setup for this insight.
```

I checked our [style guide](https://posthog.com/handbook/growth/marketing/posthog-style-guide), and it looks like we should use [bold for button names and tab names](https://posthog.com/handbook/growth/marketing/posthog-style-guide#bold-button-and-tab-names). It might be worth taking a look through the doc for other buttons and tabs for consistency.